src/include/unit_manager.h

```diff
@@ -74,6 +74,7 @@ class CUnitManager
 	std::vector<CUnit *> units;
 	std::vector<CUnit *> unitSlots;
 	std::list<CUnit *> releasedUnits;
+        std::vector<CUnit *> ptrList;
```

Comment:
inconsistent tabulation:
Check your editor configuration (we have clang-format).